CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 
 # PayPal iOS SDK Release Notes
 
+## Unreleased
+* PaymentButtons
+	* Change deprecated colors `black`, `silver`, `blue`, and `darkBlue` colors to `white`.
```

Comment:
Looks like this is indented a bit too far - we should update to the same spacing as below

Reply:
Oops sorry, I had indentation set at 4 instead of 2